app/assets/javascripts/index/directions.js

```diff
@@ -402,8 +440,10 @@ OSM.Directions = function (map) {
       }
     }
 
-    endpoints[0].setValue(params.from || "", from);
-    endpoints[1].setValue(params.to || "", to);
+    var ll_from_search_form = decodeURIComponent(location.search).match(/^\?from=([+-]?\d+(\.\d*)?)(?:\s+|\s*[/,]\s*)([+-]?\d+(\.\d*)?)$/);
```

Comment:
Does this regexp ever match anything?

"Directions from here" generate `location.search` like `?from=52.54124,13.56290&to=` which doesn't match. Do you want it to match?

Reply:
Yes, it does. It captures the situation when user inputs some coordinates into search field and then pushes "Find directions between two points" button. Earlier, this performed reverse geocoding, now it doesn't do it anymore.

Our idea behind the current solution is "if user enters the coordinates or drags pointers, leave the coordinates. If user right-clicks and selects from context menu, perform reverse geocoding".